servers/republik/.env.example

```diff
@@ -89,3 +89,10 @@ STRIPE_SECRET_KEY_REPUBLIK=
 TWILIO_API_ACCOUNT_SID=
 TWILIO_API_AUTH_TOKEN=
 TWILIO_DEFAULT_SENDER_NUMBER=
+
+# Track search requests. Default is false
+#SEARCH_TRACK=true
+
+# Run notify listener, which updates data in ElasticSearch if Postgres
+# data changes. Default is false
+#NOTIFY_LISTENER=true
```

Comment:
IMHO is `NOTIFY_LISTENER` not intuitively understandable. What do you think about `SEARCH_PG_LISTENER`. Also I think the default should be true. Why would one disable this?